prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -248,6 +248,10 @@ ssize_t rxr_pkt_post_ctrl_once(struct rxr_ep *rxr_ep, int entry_type, void *x_en
 	ssize_t err;
 	fi_addr_t addr;
 
+	/* inject path should not have additional flags for send/recv operations */
+	if (inject)
+		assert(!flags);
```

Comment:
this can be folded into the if(inject) condition later. We also need a comment explain why.

Reply:
Done.